### DISCUSSION TITLE
Question: tools.edit-lint

### DISCUSSION THREAD

**Discussion by buhtz** [bulletmark/pipxu#7] [General]
I found this section in your toml file. What is its purpose?

**Comment by bulletmark** [comment]
Converted to a discussion because this is not a bug/issue.

I have a personal tool called `edit-lint` which allows you to easily run a group of linters across your file from within your editor. Presently I use it myself (with vim) but I may publish it.

**Reply by buhtz** [reply]
Sounds interesting.

Currently I do use several linters in unit tests. Might be unusual but works. I don't like pre-commit hooks for several reasons. And I (and my team mates and contributors) also do use several machines with several IDEs and linter configs.
